README.md

```diff
@@ -130,5 +130,7 @@ and [our wiki](https://github.com/cocotb/cocotb/wiki).
 * [cocotb-based USB 1.1 test suite](https://github.com/antmicro/usb-test-suite-build) for FPGA IP, with testbenches for a variety of open source USB cores
 * [`cocotb-coverage`](https://github.com/mciepluc/cocotb-coverage), an extension for Functional Coverage and Constrained Randomization
 * [`uvm-python`](https://github.com/tpoikela/uvm-python), an almost 1:1 port of UVM 1.2 to Python
+* [`pyuvm`](https://github.com/pyuvm/pyuvm), The UVM IEEE 1800.2 specification built upon cocotb and implemented from scratch in Python.
+* [`pythonverification.com`](http://pythonverification.com), Resources for learning Python, cocotb, and pyuvm. Includes pointers to examples on GitHub.
```

Comment:
```suggestion
```
I think this comes off as an ad. I think this could absolutely be put into the Further Resources page on the Wiki or on pyuvm's README.